common/persistence/cassandra/errors.go

```diff
@@ -57,24 +57,17 @@ type (
 )
 
 func convertErrors(
-	record map[string]interface{},
```

Comment:
why the original record is removed?

according to [MapExecuteBatchCAS](https://github.com/gocql/gocql/blob/769848eae4625444c6abdabc4a67eacb117c9200/session.go#L736-L750)
record will be filled with data if CAS failed